Utils/GenerateRandomAlphanumericString.java

```diff
@@ -0,0 +1,18 @@
+package Utils;
+
+public class GenerateRandomAlphanumericString {
+    public String generateRandomAlphanumericString(int size) {
```

Comment:
Is it worth to add error handling in this method, what if input will be negative number?